engagement_interaction_tracking.md

```diff
@@ -30,6 +30,7 @@ Engagement interations are used to track how users interact and progress through
 | `share` | This category is for social media shares (i.e. Facebook, Twitter, and Pinterest).  This category has become less important today. |
 | `data_entry` | This category is for tracking data entry such as user submitted their email, name, or phone number.  This category is also seldom used today in the age of privacy. |
 | `aggregate` | This category is used for aggregating multiple interactions into one tracking event.  The `value` field is used to provide a count of the interaction.  This is seldom used. |
+| `debug` | This category is currently only used for heartbeat tracking. |
```

Comment:
I don't have full visibility into this, so this is my perspective of its usage relative to the code